src/assignment-target-type/invalid/direct.template

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/expressions/assignmenttargettype/direct-
+name: Direct assignment
+info: |
+  Direct assignment
+---*/
+$DONOTEVALUATE();
+
+function _() {
+  /*{ assignmenttarget }*/ /*{ operator }*/ /*{ value }*/;
+}
```

Comment:
Since this function is never called, it defeats the purpose of the `$DONOTEVALUATE()` mechanism: in engines that implement the error as a runtime error (instead of an early/parse error) no error is thrown because the code isn’t executed.

Could we either

a) rewrite this test without the function, or
b) keep it as-is but call the function at the end?

cc @leszekswirski

Reply:
Filed https://github.com/tc39/test262/issues/3160 for this